connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1174,6 +1309,8 @@ void Gap::on_scan_started(bool success)
 
 void Gap::on_scan_stopped(bool success)
 {
+    tr_info("Scan %s", success ? "successfully stopped" : "failed to stop");
```

Comment:
I think we can add traces bellow explaining that the address has been refreshed and advertising will restart.